### DISCUSSION TITLE
wire:model.lazy don't work on input[type=number] field

### DISCUSSION THREAD

**Discussion by ShekhSaifuddin007** [livewire/livewire#2169] [1. Help]
Is this a bug or else I don't know

I created [PlayGround](https://laravelplayground.com/#/snippets/a63a369b-4d76-4118-8f81-b3fd13f8b0bc)

You can see there

**Comment by joshhanley** [comment]
@ShekhSaifuddin007 I just ran your playground and it works fine to me 🙂 

Keep in mind that .lazy will only fire once you take focus away from the input.

So if you are clicking the arrows inside the number input and want it to change straight away, then remove the .lazy. 

Otherwise try clicking the arrows a few times then clicking somewhere else on the page or tab to another input and then it will update.

Hope this helps!
